doc/radosgw/cloud-transition.rst

```diff
@@ -75,84 +83,88 @@ Contains a list of ``acl_mappings``.
 * ``acl_mapping`` (container)
 
 Each ``acl_mapping`` structure contains ``type``, ``source_id``, and ``dest_id``. These
-will define the ACL mutation that will be done on each object. An ACL mutation allows converting source
-user id to a destination id.
+define the ACL mutation to be done on each object. An ACL mutation allows converting source
+userid to a destination userid.
 
 * ``type`` (id | email | uri)
 
-ACL type: ``id`` defines user id, ``email`` defines user by email, and ``uri`` defines user by ``uri`` (group).
+ACL type: ``id`` defines userid, ``email`` defines user by email,
+and ``uri`` defines user by ``uri`` (group).
 
 * ``source_id`` (string)
 
 ID of user in the source zone.
 
 * ``dest_id`` (string)
 
-ID of user in the destination.
+ID of user on the destination.
 
 * ``target_path`` (string)
 
-A string that defines how the target path is created. The target path specifies a prefix to which
-the source 'bucket-name/object-name' is appended. If not specified the target_path created is "rgwx-${zonegroup}-${storage-class}-cloud-bucket".
+A string that defines how the target path is constructed. The target path
+specifies a prefix to which the source bucket-name/object-name is appended.
+If not specified the ``target_path`` ``rgwx-${zonegroup}-${storage-class}-cloud-bucket``.
```

Comment:
Line 106 is missing a verb.

I would go with "If no 'target_path' is specified, the 'target_path' created is 'X'."